### PR TITLE
React week02 homework

### PR DESCRIPTION
**Pull request by Laurelinduilwen**
the basic structure and the functionality of the app is complete, will play around a bit... writing this down in the likely case of me breaking something down in the following hours :p

### CODE REVIEW

File: homework-submission/react-homework-w02/src/App.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    setTodoItems(newTodoArray);
+  };
+
+  return (
+    <div className="App">
+      <div className="todo-list">
+        {toDoItems.map((todo, index) => (
+          <React.Fragment>
+            <Todo key={index} index={index} todo={todo} />
+            <HyfButton name="Mark Complete" toggle={() => toggleTodo(index)} />
```
Comment (ArcoMul) [inline homework-submission/react-homework-w02/src/App.js]: Would be neat if you could also mark a todo item incomplete again, and that the text of the button changes to 'Mark Incomplete' once a todo item has been marked complete.

**Comment by Laurelinduilwen** [comment]
thank you for the review <3

I will try to fix the week 1 branch mistake that I made since I was rushing while submitting this and forgot to checkout to the master branch before creating the week02 branch as you pointed out >_<

File: homework-submission/react-homework-w02/src/App.js (hunk 1 above)
Comment (Laurelinduilwen) [reply homework-submission/react-homework-w02/src/App.js]: that was my initial goal with the homework as you can see with the use of the "toggle" magic word but I had a slight problem with the if statement and had to do it one sided :D

**Comment by Laurelinduilwen** [comment]
Finished the implementation of feedback  :D

**Pull request closed without merging**